src/adagrams.js

```diff
@@ -1,15 +1,153 @@
+const letterPool = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+let lettersArr = Object.keys(letterPool);
+let lettersPool = [];
+lettersArr.forEach((letter) => {
+  console.log(letter);
+  for (let i = 0; i < letterPool[letter]; i++) {
+    lettersPool.push(letter);
+  }
+});
+console.log(lettersPool);
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  let hand = [];
+  let letterPoolCopy = [...lettersPool];
```

Comment:
I see you made a copy of `lettersPool` but it doesn't seemed like it was mutated anywhere. So my question is, do you think we could have just used `lettersPool` in place of `letterPoolCopy`?

Reply:
Actually, disregard this comment! Upon further inspection I can see where you mutated it!